packages/wrangler/src/api/dev.ts

```diff
@@ -183,6 +183,7 @@ export async function unstable_dev(
 		ip: "127.0.0.1",
 		inspectorPort: options?.inspectorPort ?? 0,
 		v: undefined,
+		cwd: undefined,
```

Comment:
Not related directly to this PR... I wonder if we should actually change `StartDevOptions` to omit properties like this? Rather than having to pass in undefined?